controllers/host/host_controller.go

```diff
@@ -2151,6 +2151,23 @@ func (r *HostReconciler) Reconcile(ctx context.Context, request ctrl.Request) (r
 		return reconcile.Result{}, err
 	}
 
+	// Restore the Ptp instance status
```

Comment:
Correction -> "Restore the Host instance status".

Reply:
Changes are done